draft-ietf-core-problem-details.md

```diff
@@ -403,7 +432,7 @@ Reference:
 Initial entries in this sub-registry are as follows:
 
 | Key value | Name          |  Brief description                                                     | Reference |
-|      7807 | tunnel-7807   |  Carry RFC 7807 problem details in a Concise Problem Details data item | RFCXXXX   |
+|      7807 | tunnel-7807   |  Carry RFC 7807 problem details in a Concise Problem Details data item | RFC XXXX   |
 {: #cpdk title="Initial Entries in Custom Problem Detail Key registry"}
```

Comment:
```suggestion
{: #cpdk title="Initial Entries in the Custom Problem Detail Key registry"}
```

Reply:
Not an incorrect change, but makes we wonder: captions tend to be telegram style (not the app :-), so leaving out articles generally should be OK.